src/Service/ZGWToXxllncService.php

```diff
@@ -293,13 +385,71 @@ public function mapZGWToXxllnc(string $casetypeId, ObjectEntity $zaakTypeObject,
     }//end mapZGWToXxllnc()
 
 
+    /**
+     * Maps zgw zaaktype to xxllnc casetype.
+     *
+     * @param string|null  $casetypeId          The caseType id.
+     * @param ObjectEntity $zaakTypeObject      ZGW ZaakType object.
+     * @param array        $zaakTypeArrayObject ZGW ZaakType object.
+     *
+     * @return array caseType array.
+     */
+    public function mapZGWZaakTypeToXxllnc(?string $casetypeId = null, ObjectEntity $zaakTypeObject, array $zaakTypeArrayObject): array
```

Comment:
properties with default value should be the last values:
ObjectEntity $zaakTypeObject, array $zaakTypeArrayObject, ?string $casetypeId = null